cli/internal/cmd/upgradecheck.go

```diff
@@ -241,7 +243,7 @@ func (u *upgradeCheckCmd) upgradeCheck(cmd *cobra.Command, fetcher attestationco
 			cmd.PrintErrf(
 				"restoring Terraform workspace: %s, restore the Terraform workspace manually from %s ",
 				err,
-				filepath.Join(constants.UpgradeDir, "<upgrade-id>", constants.TerraformUpgradeBackupDir),
+				filepath.Join(constants.UpgradeDir, u.upgradeID, constants.TerraformUpgradeBackupDir),
```

Comment:
I think we wanted to get rid of the upgradeID and only expose the path (Daniel refactored that).
Can we move the upgradeDir arg in again and use that instead?